lnwallet/channel.go

```diff
@@ -2777,10 +2777,9 @@ func fundingTxIn(chanState *channeldb.OpenChannel) wire.TxIn {
 // 1. The new htlcView reflecting the current channel state.
 // 2. A Dual of the updates which have not yet been committed in
 // 'whoseCommitChain's commitment chain.
-func (lc *LightningChannel) evaluateHTLCView(view *HtlcView, ourBalance,
-	theirBalance *lnwire.MilliSatoshi, nextHeight uint64,
-	whoseCommitChain lntypes.ChannelParty) (*HtlcView,
-	lntypes.Dual[[]*paymentDescriptor], error) {
+func (lc *LightningChannel) evaluateHTLCView(view *HtlcView,
+	whoseCommitChain lntypes.ChannelParty, nextHeight uint64) (*HtlcView,
+	lntypes.Dual[[]*paymentDescriptor], lntypes.Dual[int64], error) {
```

Comment:
😍 at last no `*ourBalance` monstery

Reply:
I will not rest until the demons have been excised. 🔪👻